chart/kubeapps/crds/apprepository-crd.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: {{ include "common.capabilities.crd.apiVersion" . }}
```

Comment:
Hrm. I thought we (bitnami) were trying to move away from helm functionality which moves away from `constant-input  + chart = constant-output`? That is, using `.Capabilities` (which this does under the hood), or `lookup` functions (not used here, just as example, though I see it is used in the `_secrets.tpl`) means that the output is a function of not only the input and chart, but also the current cluster context.

Not blocking on this as I assume the decision's been made already. We do already have one use of `.Capabilities` but were trying not to add more.

Reply:
Not sure about this, but I guess the goal is to define `.Values.global.kubeVersion` so `.Capabilities` is not called. Can you confirm @juan131 ?

Reply:
Correct. Let me added the missing values and a couple of refactorizations related to `bitnami/common` more